maps/site53/site53-4.dmm

```diff
@@ -2438,8 +2435,7 @@
 	d2 = 2;
 	icon_state = "1-2"
 	},
-/turf/simulated/floor/plating,
-/area/site53/maintenance/upper_surface)
```

Comment:
wtf why did you delete upper surface maintenance

Reply:
mustve been on accident while using strongdmm, sorry bout that